### PR TITLE
chore: update example app for VertexAI

### PR DESCRIPTION
**Pull request by Lyokone**
## Description

Making the required option more obvious to be able to run on web.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Review by exaby73 — APPROVED** [review]

File: packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart [hunk 1]
```diff
@@ -137,7 +140,8 @@ class _ChatWidgetState extends State<ChatWidget> {
   );
 
   Future<void> initFirebase() async {
-    await Firebase.initializeApp();
+    // ignore: avoid_redundant_argument_values
+    await Firebase.initializeApp(options: options);
```
Comment (russellwheatley) [inline packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart:144]: does firebase need to be initialised if it doesn’t need firebase app?

**Review by russellwheatley — CHANGES_REQUESTED** [review]

File: packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart (hunk 1 above)
Comment (Lyokone) [reply packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart:144]:
```
  static FirebaseVertexAI get instance {
    return FirebaseVertexAI.instanceFor(
      app: Firebase.app(),
    );
  }
```

The FirebaseVertexAi instance is using the defaut Firebase.app so it needs to be instantiated.

Comment (russellwheatley) [reply packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart:144]: I would've thought an exception would throw without any firebase options running on web?

Comment (Lyokone) [reply packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart:144]: I'm not sure about that, the SDK might try to initialize from meta data in the header if no options are provided in the Dart side, then fail if they cannot find it in HTML header.

**Review by russellwheatley — APPROVED** [review]

**Pull request merged**